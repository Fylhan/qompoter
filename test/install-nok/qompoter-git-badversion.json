{
	"name": "test/install-nok-git-badversion",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"fylhan/posibrain": "v1.0-RC",
		"cp/fylhan": "v1.0"
	},
	"repositories": {
		"fylhan/posibrain": "https://github.com/"
	}
}
