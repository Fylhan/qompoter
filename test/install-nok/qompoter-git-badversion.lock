{
  "name": "test/install-nok-git-badversion",
  "date": "2016-11-24T22:00:33+01:00",
  "require": {
  }
}
