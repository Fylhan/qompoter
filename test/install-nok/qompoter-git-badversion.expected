Qompoter
======== install

* fylhan/posibrain v1.0-RC
  Downloading sources from Git...
  Oups, "v1.0-RC" does not exist
  Warning: error with Git, Qompoter will try downloading sources from scratch...
  Downloading sources...
  Error: no package found "https://github.com/fylhan/posibrain/v1.0-RC"
  FAILURE

FAILURE
